janus_core/calculations/md.py

```diff
@@ -267,6 +266,8 @@ def __init__(  # pylint: disable=too-many-arguments,too-many-locals,too-many-sta
         self.ensemble = ensemble
         self.seed = seed
 
+        FileNameMixin.__init__(self, struct, struct_name, file_prefix, self.ensemble)
```

Comment:
Any reason to use `self.ensemble` here?

Reply:
That is being passed in as `additional` in case `file_prefix` isn't set, it will become e.g.:
`C6H6-nvt-...`